### ISSUE TITLE
Unable to display flows in 0.13.0b1

### ISSUE THREAD

**Issue by DavidKelly999** [globus/globus-automate-client#110]
I've installed globus-automate version 0.13.0b1 via pip. When I try to display any of the flows I've created, I get the following traceback:

```
$ globus-automate flow display 34164209-7d50-4fbc-8549-d0f9db70e8cc

Traceback (most recent call last):
  File "/home/david/venvs/cfde/bin/globus-automate", line 8, in <module>
    sys.exit(app())
  File "/home/david/venvs/cfde/lib/python3.8/site-packages/typer/main.py", line 214, in __call__
    return get_command(self)(*args, **kwargs)
  File "/home/david/venvs/cfde/lib/python3.8/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/home/david/venvs/cfde/lib/python3.8/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/home/david/venvs/cfde/lib/python3.8/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/david/venvs/cfde/lib/python3.8/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/david/venvs/cfde/lib/python3.8/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/david/venvs/cfde/lib/python3.8/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/home/david/venvs/cfde/lib/python3.8/site-packages/typer/main.py", line 497, in wrapper
    return callback(**use_params)  # type: ignore
  File "/home/david/venvs/cfde/lib/python3.8/site-packages/globus_automate_client/cli/flows.py", line 526, in flow_display
    rr.render_as_result(flow_dict)
  File "/home/david/venvs/cfde/lib/python3.8/site-packages/globus_automate_client/cli/rich_helpers.py", line 435, in render_as_result
    globus_resp = GlobusHTTPResponse(resp)
  File "/home/david/venvs/cfde/lib/python3.8/site-packages/globus_sdk/response.py", line 62, in __init__
    raise ValueError("Missing client with normal response")
ValueError: Missing client with normal response
```

**Comment by kurtmckee** [comment]
@DavidKelly999 Thanks for reporting this! We'll work to get this addressed and publish another beta.

Thank you very much for testing this!

**Comment by kurtmckee** [comment]
@DavidKelly999 I reproduced the error and think that it's fixed now. The changes will be reviewed by another member of the team and then I anticipate we'll release a new beta.

**Comment by kurtmckee** [comment]
David, we've released 0.13.0b2, which should fix this issue. Please let us know if you encounter additional issues, and thank you again for testing this!
